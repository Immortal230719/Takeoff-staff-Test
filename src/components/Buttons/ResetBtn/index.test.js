import React from 'react';
import { shallow } from 'enzyme';
import { ResetBtn } from './index';

describe('ResetBtn', () => {
  it('should render correctly', () => {
    const component = shallow(<ResetBtn disabled className="center">Reset</ResetBtn>);

    expect(component.prop('disabled')).toEqual(true);
    expect(component.prop('className')).toEqual('center');
    expect(component.prop('children')).toEqual('Reset');
    expect(component).toMatchSnapshot();
  });
});
