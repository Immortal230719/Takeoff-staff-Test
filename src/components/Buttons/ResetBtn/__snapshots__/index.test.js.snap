// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetBtn should render correctly 1`] = `
<WithStyles(ForwardRef(Button))
  className="center"
  color="inherit"
  disabled={true}
  endIcon={
    <UNDEFINED
      color="inherit"
      fontSize="large"
    />
  }
  size="large"
  type="reset"
  variant="contained"
>
  Reset
</WithStyles(ForwardRef(Button))>
`;
